### DISCUSSION TITLE
Amazon Product Types

### DISCUSSION THREAD

**Discussion by nabilsarder** [amzn/selling-partner-api-models#3341] [General]
I have noticed that some product types that exist in the Product Classifier in Seller Central do not exist in the Product Type Definitions API. Those that do not exist in the API, default to the `PRODUCT` definition schema.

If the appropriate `item_type_keyword` is used with `PRODUCT` as the product type, will it be placed in the correct product type that is found in the Product Classifier?

**Comment by vaidas-lungis** [comment]
I don't think so, because `item_type_keyword` is not available in all marketplaces (for example German marketplace)

**Comment by chapmanjw** [comment]
@nabilsarder and @vaidas-lungis There are a few separate concepts to break down here:

1. Product Type -> A product type (as provided by the Product Type Definitions API) essentially refers to the data model and requirements for items using this product type (i.e. that footwear_size is a required attribute for shoes, etc.). 
2. Browse Classification -> A browse classification (such as item_type_keywords in US and recommended_browse_nodes in other marketplaces) refers to the buyer-facing hierarchy you see on the Amazon retail websites.

There is a relationship between the two, where a product type applies to one or more browse classifications. Currently, you can see this full relationship in the Browse Tree Report available in the Reports API. We will be launching an API equivalent of that in the future, but I don't have a firm timeline to provide on that. 

One last note: we are in the midst of migrating product types and making them available in the Product Type Definitions and Listings Items APIs. We are currently ~550 product types in ~12 marketplaces and are launching more each month. This does mean that we do not yet support creating listings in the Listings Items APIs if the needed product type is not launched yet. The aim is to have most product types and marketplaces by the end of the year with the remaining ones coming next yer. 

In the meantime, the Category XSDs + XML feeds are currently the only programmatic-targeted offering that covers all product types today.

**Reply by nkev** [reply]
@chapmanjw Do you know when the GET_XML_BROWSE_TREE_DATA report is going to be available for vendors? 

More specifically, Is there a way for Australian Vendors to get a list of  `product_type` and `recommended_browse_nodes` values so we can change `product_type` and  `recommended_browse_nodes` for products using the Listings API? We currently cannot change these in any way, Vendor Central or API!

**Reply by ciscowu** [reply]
hi. Is there an API that can query the relationship between the two?
